Formula/sleepwatcher.rb

```diff
@@ -1,8 +1,8 @@
 class Sleepwatcher < Formula
-  desc "Monitors sleep, wakeup, and idleness of a Mac"
+  desc "Monitors sleep, wakeup, and idleness of a Mac. 64-bit as of v2.2.1"
```

Comment:
Please revert `desc` to its previous state

Reply:
Done.